src/command/current.rs

```diff
@@ -90,11 +98,14 @@ Options:
                     );
                 }
 
-                Ok(any)
+                any
             }
         };
         session.add_event_end(ActivityKind::Current, ExitCode::Success);
-        result
+        if !result {
+            throw!(NoVersionsFoundError);
```

Comment:
It doesn't seem right to me that not having a currently selected node version is an error. But since command syntax is still up in the air, I'm fine with this for now.

Reply:
Agreed, and we don't need to resolve this right this second, but just to explain the intuition I had here, it was so that you could use e.g. `notion current > /dev/null` in shell scripting to determine if there's a current Node version. IOW, a CLI "error" is also just a "falsey value" in the shell.